src/FSharpPlus/Extensions/Result.fs

```diff
@@ -68,7 +68,13 @@ module Result =
         with e -> Error e
 
     /// Gets the 'Ok' value. If it's an 'Error' this function will throw an exception.
-    let get (source: Result<'T,'Error>) = match source with Ok x -> x | _ -> invalidArg "source" "Result value was Error"
+    let get (source: Result<'T,'Error>) =
```

Comment:
I don't get why we get the warning around `The type variable 'Error has been constrained to be type 'obj'`

Reply:
Oh, that's because the function is not inline, so the use of `string` over a generic type constraint to `obj`.

I think this was somehow fixed in F#5 , for the moment maybe we should either make it inline or use the boxed variable.
Performance is not important here as the application is about to stop at that point.